cont.c

```diff
@@ -28,6 +28,9 @@
 #include "internal/scheduler.h"
 #include "mjit.h"
 #include "vm_core.h"
+#include "vm_callinfo.h"
+#include "vm_insnhelper.h"
+
```

Comment:
Apply https://github.com/ruby/ruby/pull/3766/checks?check_run_id=1401035342

Reply:
Oh, thanks! When I run `make all golf` followed by `ruby tool/update-deps --fix` locally, it doesn't change anything. But I applied the diff manually.